pkg/client/retrieve.go

```diff
@@ -34,10 +32,19 @@ import (
 	"k8s.io/client-go/tools/remotecommand"
 )
 
-func (c *SonobuoyClient) RetrieveResults(cfg *RetrieveConfig) (io.Reader, error) {
+var tarCommand = []string{
```

Comment:
why do we need to wrap this call with `bash -c`?

Reply:
because the `sh` execution doesn't handle the `*` globbing

Reply:
were you running into issues with the existing code?

Reply:
Yes. If you ran the command before sonobuoy had finished running, it would tar up and retrieve a temporary directory that didn't have anything useful in it. Users had to detect if the untarred file ended in .tar.gz to determine if it was successful.